client/devdocs/gutenberg/README.md

```diff
@@ -0,0 +1,4 @@
+Gutenberg Blocks
+====
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent ut arcu et mauris fermentum tempus. In vel lacinia lorem, nec accumsan tortor. Nulla non tellus interdum, vestibulum nisl sed, placerat metus.
```

Comment:
obviously this needs to be filled out before merging 😉

Reply:
my plan was to do that in a separate PR as I thought it was ok to have "lorem ipsum" content if that is behind a feature flag.

anyway, I have just added a proper documentation in order to get rid of the lorem ipsum ;)

Reply:
I've just realized a problem we have with this: all the README.md files can be accessed even if the feature flag is not active.

I'll check later how can that be improved, so the documentation is not visible if the feature flag is not enabled

Reply:
it's not about feature-flagging it. it's about committing code that really has no purpose. it's fine to not include it, it's fine to take the time to write it, but it's not really a Good Thing™️ to ship code that's dead at the get-go.

my philosophy is that code that merges is permanent. there is no such thing as temporary code or "I'll get to it soon."